src/plugins/orangehrmAdminPlugin/Service/LocalizationService.php

```diff
@@ -128,4 +130,53 @@ public function getLanguagesArray(I18NLanguageSearchFilterParams $i18NLanguageSe
         $languages = $this->searchLanguages($i18NLanguageSearchParams);
         return $this->getNormalizerService()->normalizeArray(I18NLanguageModel::class, $languages);
     }
+
+    /**
+     * @param int $languageId
+     * @param array $rows
```

Comment:
```suggestion
     * @param array $rows e.g. [['langStringId'=> 1, 'translatedValue' => 'Employee'], ['langStringId'=> 2, 'translatedValue' => 'Admin']]
```
add an example like this